docs/reference/contributing/guidelines/ci.md

```diff
@@ -0,0 +1,55 @@
+## Continuous integration
+
+Continuous integration (CI) means mainly automatic testing for pull requests
+
+### Travis CI
+
+In Mbed OS [Travis CI](https://travis-ci.org/ARMmbed/mbed-os) is used as primary automatic testing and checking run environment.
+
+Travis configuration is located in [.travis.yml](https://github.com/ARMmbed/mbed-os/blob/master/.travis.yml) file in Mbed OS root directory. Mbed OS use public travis so test results are publicly available and there are public [documentation available](https://docs.travis-ci.com/).
```

Comment:
Travis configuration is located in  [.travis.yml](...)=> The Travis configuration is located in the  [.travis.yml](...) file